lib/src/core/provider/power_off_provider.dart

```diff
@@ -31,6 +34,16 @@ class PowerOffProvider with ChangeNotifier {
 
   final List<TimetableModel> _timeTableItems = [];
 
+  double widgetChildSize = 0;
+
+  double widgetParentSize = 0;
+
+  ScrollController? headersListScrollController;
+
+  int? _index;
+
```

Comment:
I think it would be better to use undefined index, so it would not be nullable. e.g. `int _index = -1`